modules/ppcp-axo/src/AxoModule.php

```diff
@@ -367,6 +369,56 @@ static function ( $methods ) {
 			}
 		);
 
+		// Trigger 3D Secure verification.
+		add_filter(
+			'ppcp_create_order_request_body_data',
+			function( array $data, string $payment_method ) use ( $c ): array {
+				if ( ! $c->get( 'axo.uk.enabled' ) ) {
+					return $data;
+				}
+
+				$logger = $c->get( 'woocommerce.logger.woocommerce' );
+				assert( $logger instanceof LoggerInterface );
+
+				if ( $payment_method !== AxoGateway::ID ) {
+					return $data;
+				}
+
+				$settings_model = $c->get( 'settings.data.settings' );
+				assert( $settings_model instanceof SettingsModel );
+
+				$three_d_secure = $settings_model->get_three_d_secure_enum();
+
+				if (
+					$three_d_secure === 'SCA_ALWAYS'
+					|| $three_d_secure === 'SCA_WHEN_REQUIRED'
+				) {
+					$data['payment_source']['card']->attributes = array(
```

Comment:
Are we sure that the `$data['payment_source']` and `$data['payment_source']['card']` exist?